archive/100-theorems-list/inverse-triangle-sum.lean

```diff
@@ -0,0 +1,87 @@
+import tactic
+import algebra.big_operators
+import data.real.basic
+
+lemma range_insert (n : ℕ) :
+insert n (finset.range n) = finset.range (n + 1) :=
```

Comment:
isn't this a `simp` lemma?

Reply:
It does exist in mathlib as finset.range_succ. I'm not able to replace these `rw`s with `simp`s, though.